shared/panels/_flows/reward-flow/_panels/single-contribution-validation/single-contribution-validation.tsx

```diff
@@ -131,7 +131,7 @@ function Content() {
           translate={{
             token: "common:reward",
           }}
-          isDisabled={isCreatingRewards || amountNumber <= 0}
+          isDisabled={isCreatingRewards || amountNumber <= 0 || (budget && amountNumber > budget.amount)}
```

Comment:
Maybe do a function isRewardDisabled (something like that) to simplify the props?